libshaderc/include/shaderc.h

```diff
@@ -63,6 +63,15 @@ typedef enum {
   shaderc_profile_es,
 } shaderc_profile;
 
+// Used in the result module (shaderc_spv_module) to tell the type of errors
+// generated during compilation.
+typedef enum {
+  shaderc_compilation_result_success,
+  shaderc_compilation_result_failed_invalid_stage,  // error in the deduction of shader stage
+  shaderc_compilation_result_compilation_failed,
+  shaderc_compilation_result_null_result_module,
+} shaderc_compilation_result;
```

Comment:
Old GetSuccess() (the cpp api) checks the existence of a result module, and return false if there isn't. So need to add this one of the compilation results.